lib/ansible/modules/monitoring/honeybadger_deployment.py

```diff
@@ -137,7 +137,7 @@ def main():
         e = get_exception()
         module.fail_json(msg='Unable to notify Honeybadger: %s' % e)
     else:
-        if info['status'] == 200:
+        if info['status'] == 201:
```

Comment:
As I read your code HTTP 200 brings module faillure, I'm not using Honeybadger but it seems strange to me.